test/test_collection.py

```diff
@@ -213,5 +213,19 @@ def test_cli_tool(self):
         self.assertNotEqual(res.stdout, "")
         self.assertTrue(res.returncode == 0)
 
+    def test_imports(self):
+        try:
+            from pymongoexplain import ExplainCollection
+        except:
+            print("Could not import!")
+            assert False
+        try:
+            from pymongoexplain import ExplainableCollection
+        except:
+            print("Could not import!")
+            assert False
+        assert ExplainableCollection == ExplainCollection
```

Comment:
You can remove the try/excepts and use `self.assertEqual` instead of bare assert.

Reply:
Done.